src/microbe_stage/MicrobeCamera.cs

```diff
@@ -144,6 +146,16 @@ public void SetBackground(Background background)
         {
             materialToUpdate.SetShaderParam($"layer{i:n0}", GD.Load<Texture>(background.Textures[i]));
         }
+
+        if (BackgroundParticles != null)
```

Comment:
I don't particularly like that the particles are used in the editor.
Making an [Export] bool variable in the camera class that disables them would be nice, and then setting that true in the editor scene for the camera. (the same way the default zoom is different).